doc/changes/changes49.xml

```diff
@@ -0,0 +1,1196 @@
+required dependency, not just an optional one.
+</Item>
+<Item>
+A number of improvements have been made to <Ref BookName="ref" Oper="Random"/>. 
+These may lead to different sequences of numbers being created.
+On the up side, many more methods for <Ref BookName="ref" Oper="Random"/>
+(and other <C>RandomXYZ</C> operations) now optionally take an explicit
+<Ref BookName="ref" Oper="RandomSource"/> as first argument
+(but not yet all: help with this is welcome). Some relevant pull requests:
+<List>
+<Item>
+    Allow creating random permutations using a random source
+   (<URL Text="#1165">https://github.com/gap-system/gap/pull/1165</URL>)
+</Item>
+<Item>
+    Let more <Ref BookName="ref" Oper="Random"/> methods use an alternative source
+    (<URL Text="#1168">https://github.com/gap-system/gap/pull/1168</URL>)
+</Item>
+<Item>
+    Help <Ref BookName="ref" Oper="Random"/> methods to
+    use <Ref BookName="ref" Oper="RandomSource"/>
+    (<URL Text="#810">https://github.com/gap-system/gap/pull/810</URL>)
+</Item>
+<Item>
+    Remove uses of old random generator
+    (<URL Text="#808">https://github.com/gap-system/gap/pull/808</URL>)
+</Item>
+<Item>
+    Fix <Ref BookName="ref" Oper="Random"/> on long (>2^28) lists
+    (<URL Text="#781">https://github.com/gap-system/gap/pull/781</URL>)
+</Item>
+<Item>
+    Fix <Ref BookName="ref" Func="RandomUnimodularMat"/>
+    (<URL Text="#1511">https://github.com/gap-system/gap/pull/1511</URL>)
+</Item>
+<Item>
+    Use <Ref BookName="ref" Oper="RandomSource"/> in a few more places
+    (<URL Text="#1599">https://github.com/gap-system/gap/pull/1599</URL>)
+</Item>
+</List>
+</Item>
+<Item>
+The output and behaviour of the profiling system has been substantially improved
+<List>
+<Item>
+    Make profiling correctly handle the same file being opened multiple times
+    (<URL Text="#1069">https://github.com/gap-system/gap/pull/1069</URL>)
+</Item>
+<Item>
+    Do not profile the <C>return</C> statements
+    (<Ref BookName="ref" Label="return"/>) inserted into the end of functions
+    (<URL Text="#1073">https://github.com/gap-system/gap/pull/1073</URL>)
+</Item>
+<Item>
+    Ensure we reset <C>OutputtedFilenameList</C>
+    in profiling when a workspace is restored
+    (<URL Text="#1164">https://github.com/gap-system/gap/pull/1164</URL>)
+</Item>
+<Item>
+    Better tracking of amounts of memory allocated and time spent
+    in the garbage collector
+    (<URL Text="#1806">https://github.com/gap-system/gap/pull/1806</URL>)
+</Item>
+<Item>
+    Allow profiling of memory usage
+    (<URL Text="#1808">https://github.com/gap-system/gap/pull/1808</URL>)
+</Item>
+<Item>
+    Remove profiling limit on files with &lt;= 2^16 lines
+    (<URL Text="#1913">https://github.com/gap-system/gap/pull/1913</URL>)
+</Item>
+</List>
+</Item>
+<Item>
+In many cases &GAP; now outputs the filename and location of functions in
+helpful places, e.g. in backtraces or when displaying compiled functions.
```

Comment:
I wonder if everybody will know what a "backtrace" is -- perhaps say "in error messages" ? Or "in backtraces printed after an error occurred"? Or so?